src/entities/email_campaign.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+import { CAMPAIGN_TYPE } from 'src/utils/constants';
+import { BaseEntity } from './base.entity';
+
+@Entity({ name: 'email_campaign' })
+export class EmailCampaignEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
+  campaignType: CAMPAIGN_TYPE;
+
+  @Column({ unique: true })
+  campaignInfo: string;
+
+  @Column()
+  email: string;
+
+  @Column({ type: 'timestamptz', nullable: true })
```

Comment:
hmm - just thinking whether this wants to be nullable. Not a huge deal but wondering what the use case is.

Reply:
Good catch, have changed 👍